docs/content/tutorial/resize-cluster.md

```diff
@@ -13,20 +13,26 @@ Let's dive in.
 
 ## Resize Memory and CPU
 
-Memory and CPU resources are an important component for vertically scaling your Postgres cluster. Couple with [tweaks to your Postgres configuration file]({{< relref "./customize-cluster.md" >}}), allowing your cluster to have more memory and CPU allotted to it can help it to perform better under load.
+Memory and CPU resources are an important component for vertically scaling your Postgres cluster.
+Couple with [tweaks to your Postgres configuration file]({{< relref "./customize-cluster.md" >}}),
+allowing your cluster to have more memory and CPU allotted to it can help it to perform better under load.
 
-It's important for instances in the same high availability set to have the same resources. PGO lets you adjust CPU and memory within the `resources` sections of the `postgresclusters.postgres-operator.crunchydata.com` custom resource. These include:
+It's important for instances in the same high availability set to have the same resources.
+PGO lets you adjust CPU and memory within the `resources` sections of the `postgresclusters.postgres-operator.crunchydata.com` custom resource. These include:
 
-- `spec.instances.resources` section, which sets the resource values for the PostgreSQL container, as well as any init containers in the associated pod and containers created by the `pgDataVolume` and `pgWALVolume` [data migration jobs]({{< relref "guides/data-migration.md" >}}).
-- `spec.instances.sidecars.replicacertcopy.resources` section, which sets the resources for the `replica-cert-copy` sidecar container.
+- `spec.instances.resources` section, which sets the resource values for the PostgreSQL container,
+  as well as any init containers in the associated pod and containers created by the `pgDataVolume` and `pgWALVolume` [data migration jobs]({{< relref "guides/data-migration.md" >}}).
+- `spec.instances.sidecars.replicaCertCopy.resources` section, which sets the resources for the `replica-cert-copy` sidecar container.
 - `spec.monitoring.pgmonitor.exporter.resources` section, which sets the resources for the `exporter` sidecar container.
-- `spec.backups.pgbackrest.repoHost.resources` section, which sets the resources for the pgBackRest repo host container, as well as any init containers in the associated pod and containers created by the `pgBackRestVolume` [data migration job]({{< relref "guides/data-migration.md" >}}).
+- `spec.backups.pgbackrest.repoHost.resources` section, which sets the resources for the pgBackRest repo host container,
+  as well as any init containers in the associated pod and containers created by the `pgBackRestVolume` [data migration job]({{< relref "guides/data-migration.md" >}}).
```

Comment:
❓ Does Hugo correctly format an item in a list if it's split over two lines?

Reply:
It does in v0.61, at least.

<img width="297" alt="Screen Shot 2022-02-07 at 11 56 18 AM" src="https://user-images.githubusercontent.com/106560/152845153-818121fa-296c-4263-9db5-f4142eee87d7.png">